.github/workflows/packages.yml

```diff
@@ -12,6 +12,8 @@ jobs:
       - uses: actions/checkout@v4
 
       - uses: bufbuild/buf-setup-action@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@moritzzimmer I added this to all steps. reason:
<img width="1401" alt="Bildschirmfoto 2024-07-09 um 11 39 09" src="https://github.com/stroeer/tapir/assets/733158/62ff61d2-a5fc-444d-b507-05f1ed9a4f4a">